uint8 event_type
uint8 event_id
string query_message
JUDPMessage create_message